### PR TITLE
Dont log full GCS errors because it's noisy given implied applications.

### PR DESCRIPTION
**Pull request by ewiseblatt**
Also, delete (purge) old versions of the timestamp files since they are particularly
volatile and it doesnt make sense version them.

### CODE REVIEW

**Comment by duftler** [comment]
Looks like a unit test failed.

**Comment by ewiseblatt** [comment]
fixed

**Comment by ewiseblatt** [comment]
ping

File: front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/BucketDAO.java [hunk 1]
```diff
@@ -214,6 +217,13 @@ protected void refresh() {
 
     Set<T>  result = resultMap.values().stream().collect(Collectors.toSet());
```
Comment (duftler) [inline front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/BucketDAO.java]: Extraneous space just before `result`.

File: front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/BucketDAO.java [hunk 2]
```diff
@@ -214,6 +217,13 @@ protected void refresh() {
...
     this.lastRefreshedTime = refreshTime;
+
+    int result_size  = result.size();
```
Comment (duftler) [inline front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/BucketDAO.java]: Extraneous space just before `=`.

File: front50-gcs/src/test/groovy/com/netflix/spinnaker/front50/model/GoogleStorageServiceSpec.groovy [hunk 3]
```diff
@@ -179,6 +179,8 @@ class GoogleStorageServiceSpec extends Specification {
   def "storeObject"() {
     given:
       Storage.Objects.Insert mockInsertObject = Mock(Storage.Objects.Insert)
+      Storage.Objects.List mockListObjects = Mock(Storage.Objects.List)
+     com.google.api.services.storage.model.Objects objects = new com.google.api.services.storage.model.Objects()
```
Comment (duftler) [inline front50-gcs/src/test/groovy/com/netflix/spinnaker/front50/model/GoogleStorageServiceSpec.groovy]: Indentation is off here.

**Comment by duftler** [comment]
LGTM once minor formatting comments are addressed.

**Pull request merged**